OrganisationSample/src/main/java/models/OrganisationEmployee.java

```diff
@@ -0,0 +1,23 @@
+package models;
+
+import lombok.Data;
+import utils.DepartmentNameUtility;
+import utils.EmployeeTypeUtility;
+
+@Data
+public class OrganisationEmployee {
+
+    private String empID, empName,empEmailID,empContactNo;
+    private DepartmentNameUtility departmentType;
```

Comment:
rename DepartmentNameUtility to DepartmentName. Makes more sense for enum, besides it is not performing any action.